functions/Get-DbaStartupParameter.ps1

```diff
@@ -164,7 +178,8 @@ Logs in to WMI using the ad\sqladmin credential and gathers simplified informati
 			}
 			catch
 			{
-				Write-Warning "$servername`: $_ "
+			#	Write-Message -Level Warning -Message "$servername`: $_ " -silent $silent
+			write-warning "caught error $_"
```

Comment:
That's a removed line I believe, the next line writes the error which will be $_.